recipes/pythia-datasets/LICENSE.txt

```diff
@@ -0,0 +1,201 @@
+Apache License
```

Comment:
Missed this the first time. Since `LICENSE` already is in the PyPI tarball, just point to that and remove this file from the PR.

Reply:
Thanks @dopplershift, I was confused about this. Is it enough to just set 
```
license_file: LICENSE
```
or does a more specific path need to be specified?